Gemfile

```diff
@@ -77,6 +77,7 @@ gem "git", git: "https://github.com/fastlane/ruby-git", require: false # Interac
 
 group :test, :development do
   gem "coveralls"
+  gem "foreman"
```

Comment:
I can move this out of the `:test, :development` groups if you think that's better too.